eng/Versions.props

```diff
@@ -15,21 +15,21 @@
 
     <!-- .NET Runtime product dependencies -->
     <MicrosoftBclAsyncInterfacesVersion>6.0.0</MicrosoftBclAsyncInterfacesVersion>
-    <MicrosoftExtensionsVersion>2.1.0</MicrosoftExtensionsVersion>
+    <MicrosoftExtensionsVersion>6.0.0</MicrosoftExtensionsVersion>
     <MicrosoftExtensionsDependencyInjectionVersion>6.0.0</MicrosoftExtensionsDependencyInjectionVersion>
-    <SystemBuffersVersion>4.5.0</SystemBuffersVersion>
-    <SystemCodeDomVersion>4.5.0</SystemCodeDomVersion>
-    <SystemCollectionsImmutableVersion>1.5.0</SystemCollectionsImmutableVersion>
-    <SystemDrawingCommonVersion>4.5.0</SystemDrawingCommonVersion>
-    <SystemIOFileSystemAccessControl>4.5.0</SystemIOFileSystemAccessControl>
-    <SystemMemoryVersion>4.5.3</SystemMemoryVersion>
-    <SystemReflectionEmitLightweightVersion>4.3.0</SystemReflectionEmitLightweightVersion>
-    <SystemReflectionEmitVersion>4.3.0</SystemReflectionEmitVersion>
+    <SystemBuffersVersion>4.5.1</SystemBuffersVersion>
+    <SystemCodeDomVersion>6.0.0</SystemCodeDomVersion>
+    <SystemCollectionsImmutableVersion>6.0.0</SystemCollectionsImmutableVersion>
+    <SystemDrawingCommonVersion>5.0.3</SystemDrawingCommonVersion>
+    <SystemIOFileSystemAccessControl>5.0.0</SystemIOFileSystemAccessControl>
+    <SystemMemoryVersion>4.5.4</SystemMemoryVersion>
+    <SystemReflectionEmitLightweightVersion>4.7.0</SystemReflectionEmitLightweightVersion>
+    <SystemReflectionEmitVersion>4.7.0</SystemReflectionEmitVersion>
     <SystemRuntimeCompilerServicesUnsafeVersion>6.0.0</SystemRuntimeCompilerServicesUnsafeVersion>
```

Comment:
@ericstj  I know you didn't change this version but now the net fx tests are failing cause it can't find an older version of this. https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-machinelearning-refs-pull-6168-merge-92a6c92afaf64e6199/Microsoft.ML.CpuMath.UnitTests/1/console.07abc2b7.log?sv=2019-07-07&se=2022-05-17T17%3A14%3A13Z&sr=c&sp=rl&sig=d2SykNg38%2F8LT3JygfLEnE73p0TL7RCqKCSc%2F2QuJxE%3D. Any ideas?

Reply:
Yeah, need binding redirects.  I'll get back to this PR once I have some time to work through the failures.